src/fonts/OTF/TeX/makeFF

```diff
@@ -1847,11 +1828,7 @@ $extra{'Main-Regular'} = {
   ],
 
   diacriticals => [
-    'Select(0u2D9)', 'SetRBearing(111,1)',    # \dot
-    'Select(0uB0)',  'SetRBearing(-125,1)',   # ring above
-    'Select(0u2DA)', 'SetRBearing(-125,1)',   # ring above
-    'Select(0u307)', 'SetRBearing(111,1)',    # \dot (combining)
-    'Select(0u30A)', 'SetRBearing(-125,1)',   # ring above (combining)
+    'Select(0uB0)',  'SetRBearing(-125,1)',   # \degree
```

Comment:
So if I understand correctly, the code above here is changing the mapping of diacritics so that they all map to the Unicode Spacing Modifiers block instead of the Combining Diacritics block.

And the changes below are tweaking the fonts metrics to remove special cases for the ring and dot diacritics (which were the ones that were coming out wrong) and to add a special case for the degree symbol.

Looks good to me.

Reply:
That's correct.  symbols.js in Khan/KaTeX wasn't using the combing characters but was using spacing modifiers so this change seemed more natural.  And the degree symbol is the only one that needs a horizontal shift.